frontend/packages/console-shared/src/hooks/useUserSettings.ts

```diff
@@ -85,16 +85,12 @@ export const useUserSettings = <T>(
     if (isLocalStorage) {
       return;
     }
-    if (cfLoadError || (!cfData && cfLoaded)) {
+    if (cfLoadError?.response?.status === 404 || (!cfData && cfLoaded)) {
```

Comment:
I would expect the condition after the `||` to evaluate to `true` on other load errors, so I'm not sure if checking for 404 will change the behavior.